### ISSUE TITLE
TFLite's DepthwiseConv2D is broken with GpuDelegate for 1-channel input

### ISSUE THREAD

**Issue by bioothod** [tensorflow/tensorflow#30311]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): tf docker images
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: oneplus5, pixel3, galaxy s10
- TensorFlow installed from (source or binary): various
- TensorFlow version (use command below):  1.14.0, 1.15.0-dev20190628, r1.14 source build
- Python version: 3.x
- Bazel version (if compiling from source): 0.25.2
- GCC/Compiler version (if compiling from source): 7.4.0
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the current behavior**
When using supersimple network consisting of NHWC input, single tf.layers.conv2d() layer (constant initialized) and reduce_sum output, output for C=3 is the same for CPU (host tf and tflite and android tflite) and android GpuDelegate are kind of the same (less than 2% difference). When using C=1 CPU host (tf and tflite) and android on CPU are kind of the same (less than 10% difference), but android GpuDelegate produces drastically different output.

**Describe the expected behavior**
produce kind of the same results for the same models (no matter C=1 or C=3) for CPU host tf and tflite and android tflite on CPU and GpuDelegate.

**Code to reproduce the issue**
Attached archive with broken model generated with 1.15.0-dev20190628 tflite converted with default options (no properties changed) and script to create model and tflite file itself.

**Other info / logs**
I tested tflite both from nightly build from jcenter and local builds from r1.14 branch.
Here are the results when providing 1.224.224.1 image filled with 0.1 value:
cpu host tf: 5677536.5,
cpu host tflite: 6028237.0 (6.2% difference)
android cpu: 6028237
android gpudelegate: 44336.1

[test_model.tar.gz](https://github.com/tensorflow/tensorflow/files/3350494/test_model.tar.gz)

Also, tflite converted models seems to always run on cpu despite using gpudelegate, for example this model runs for hundreds of milliseconds no matter whether cpu or gpudelegate is used, while stock mobilenet_v1 (no quantization) runs ~80ms per inference on this device when using gpudelegate and 150-300ms on cpu.
I will fill another bug report for this though

**Comment by bioothod** [comment]
Attached model with pure conv2d layers and incorrect output results on cpu and gpudelegate, probably related to some fp16 overflow? I do not use precision loss option in gpudelegate, it was created with default parameters.

output node is [1][1]float at 0-th offset
android cpu: 244059422720
android gpudelegate: 1572096

[test_conv2d_3channels_conv2d_possible_overflow.tflite.gz](https://github.com/tensorflow/tensorflow/files/3356153/test_conv2d_3channels_conv2d_possible_overflow.tflite.gz)

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
